src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTestForFailingNotification.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.tasks;
+
+import com.microsoft.azure.storage.blob.CloudBlockBlob;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.bulkscanprocessor.config.IntegrationTest;
+import uk.gov.hmcts.reform.bulkscanprocessor.model.out.msg.ErrorCode;
+import uk.gov.hmcts.reform.bulkscanprocessor.model.out.msg.ErrorMsg;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.verify;
+import static uk.gov.hmcts.reform.bulkscanprocessor.helper.DirectoryZipper.zipDir;
+import static uk.gov.hmcts.reform.bulkscanprocessor.model.common.Event.FILE_VALIDATION_FAILURE;
+import static uk.gov.hmcts.reform.bulkscanprocessor.model.common.Event.ZIPFILE_PROCESSING_STARTED;
+
+@IntegrationTest
+@RunWith(SpringRunner.class)
+@TestPropertySource(properties = {
+    "containers.mappings[0].container=bulkscan",
+    "containers.mappings[0].jurisdiction=BULKSCAN",
+    "containers.mappings[0].poBox=BULKSCANPO",
+    "containers.mappings[0].paymentsEnabled=false",
+    "containers.mappings[0].enabled=false"
+})
+public class BlobProcessorTaskTestForFailingNotification extends ProcessorTestSuite<BlobProcessorTask> {
+
+    @Before
+    public void setUp() throws Exception {
+        super.setUp();
+
+        processor = new BlobProcessorTask(
+            blobManager,
+            envelopeProcessor,
+            zipFileProcessor,
+            containerMappings,
+            ocrValidator,
+            serviceBusHelper,
+            paymentsEnabled
+        );
+    }
+
+    @Test
+    public void should_abandon_rejected_file_when_sending_error_notification_fails() throws Exception {
```

Comment:
The file is not rejected yet. How about `should_abandon_processing_invalid_file...`?

Reply:
ok